src/components/HelloComponents.js

```diff
@@ -0,0 +1,17 @@
+import { Component } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Planthor-Team/Planthor_ClientFrontEndWebApp/pullRequest?prid=7095961)
```suggestion
import { Component } from "react";
```